source/manage/cloud-billing.rst

```diff
@@ -56,7 +56,7 @@ Invoices appear immediately after the billing period ends.
 
 Billing periods begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month, at which time subscription costs are calculated, and a credit card charge is automatically attempted. It may take up to a day for our billing systems to calculate, charge your credit card, and issue your invoice. Once charges are completed, the invoice will show as *Paid*. The invoice is a downloadable PDF file.
```

Comment:
Under "Subscription" section, let's add a note that you can also view any workspace limits on this page, e.g.

```
The subscription screen outlines the details of your Mattermost Cloud account, including the cost per user per month and the number of users currently registered in your workspace. You can also find a summary of your last invoice in the same page, and any <workspace limits><link to workspace limits doc> set for your subscription.
```